cabal-testsuite/README.md

```diff
@@ -4,11 +4,12 @@ frameworks.
 How to run
 ----------
 
-1. Build `cabal-tests` (`cabal build cabal-tests`)
+1. Build `cabal-testsuite` (`cabal build cabal-testsuite`)
 2. Run the `cabal-tests` executable. It will scan for all tests
    in your current directory and subdirectories and run them.
-   To run a specific set of tests, use `cabal-tests PATH ...`.  You can
-   control parallelism using the `-j` flag.
+   To run a specific set of tests, use `cabal-tests PATH ...`.
+   (e.g. `cabal exec -- cabal-tests cabal-testsuite/PackageTests/BuildTargets/UseLocalPackage/use-local-version-of-package.ts.hs`)
```

Comment:
The `exec` is to automatically find the full path of the exe? I wonder what's the best idiom for that. Another is `$(cabal list-bin exe:cabal-tests)` and I'm sure there are others. Which one works best for you? Doesn't `exec` also do some potentially disruptive things behind the scenes (I don't remember)?

Reply:
Yes, exec finds the path. I use Stack mostly; in Cabal projects, is it common for people to "just know" they have to find the test executable somehow?

Or maybe the docs should reference `cabal test` instead?

Reply:
`cabal run cabal-tests` worked fine for me. I assume `cabal test cabal-tests` would too. I don't see why we would advertise such a niche command as `cabal exec`.

Reply:
Sure, again, I don't use Cabal, so I didn't know what command to use. I don't particularly care which command to document, but we should document one of them

Reply:
Fully agree. Thanks a lot for taking the time to prepare a patch!

Reply:
Fixed: f972cf89339d2265676e32c4628975c24b63f726